go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/spiffe/spire-controller-manager
 
-go 1.21
+go 1.22
```

Comment:
We should probably specify the full version here (allowed since go1.21).

Reply:
Done.